src/egon/data/datasets/gas_grid.py

```diff
@@ -847,93 +863,25 @@ def insert_gas_data():
     This function inserts data into the database and has no return.
 
     """
-    download_SciGRID_gas_data()
-
-    gas_nodes_list = define_gas_nodes_list()
+    s = config.settings()["egon-data"]["--scenarios"]
+    scenarios = []
+    if "eGon2035" in s:
+        scenarios.append("eGon2035")
+    if "eGon100RE" in s:
+        scenarios.append("eGon100RE")
 
-    insert_CH4_nodes_list(gas_nodes_list)
-    abroad_gas_nodes_list = insert_gas_buses_abroad()
+    for scn_name in scenarios:
+        download_SciGRID_gas_data()

```

Comment:
Is it needed to download the data for each scenario?

Reply:
Right. We need to execute download_SciGRID_gas_data() just once. Change applied in f46ed0c800ef188c0821a7f7e34c1fccd7d927d1